ziti-springboot-client/src/main/java/org/openziti/springboot/client/web/config/EnableZitiHttpClient.java

```diff
@@ -28,7 +28,7 @@
 @Target(ElementType.TYPE)
 @Documented
 @Import({ZitiHttpClientConfiguration.class})
-@ConditionalOnProperty(value = "spring.ziti.httpclient.enabled", havingValue = "true", matchIfMissing = true)
+@ConditionalOnProperty(value = "spring.ziti.client.enabled", havingValue = "true", matchIfMissing = true)
```

Comment:
was there a necessity to change this? wouldn't this affect previous SDK users?

Reply:
On second thought, is it necessary to keep the old property? This is a release with a version bump (not a patch) so users can/should expect changes.

Reply:
you need to update sample as well